packages/metrics/src/data_providers/CsvDataProvider.test.ts

```
@@ -73,6 +79,24 @@ describe("CsvDataProvider", () => {
     ).rejects.toThrow("CSV must not be empty.");
   });
 
+  test("fetchData() fails if metric is missing a 'column' property.", async () => {
```

This error was already present and worked, but I added a test since people complained that it was failing "silently" during the hackathon.  In reality, the error was being surfaced by the `useData()` hooks, but they were not handling it.  So #398 added logging to make it harder to ignore errors from the hooks.